### PR TITLE
Mustafa/Week3

### PR DESCRIPTION
**Pull request by mayqf**

### CODE REVIEW

**Comment by ArcoMul** [comment]
Hi @mayqf for two reasons I can highly recommend trying Netlify 😁 One it will be good practice to try it once and see how powerful it is. Second one, out of self interest, it makes it easier for me to check your homework (so that I don't have to install everything locally).

**Comment by mayqf** [comment]
m

**Comment by mayqf** [comment]
,,

**Comment by mayqf** [comment]
,

**Comment by mayqf** [comment]
,

**Comment by mayqf** [comment]
many thanks for your review. Here you go!       https://goofy-swirles-b5cf0b.netlify.com/

File: homework-submission/react-hyf/react-hyf-client/src/components/homework3.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        }
+        else if (error) {
+            return <div>Error: {error.message}</div>;
+        }    
+        return (
+          <React.Fragment>
+            <div className="week3">
+              <Image image={users[userIndex].photo} />
+              <FullName name={users[userIndex].name} surname={users[userIndex].surname} />
+              <Users user={users[userIndex]} />
```
Comment (ArcoMul) [inline homework-submission/react-hyf/react-hyf-client/src/components/homework3.js]: Would look nicer if you'd destructure the properties out of `users[userIndex]`

**Pull request closed without merging**